spring-auto-restdocs-docs/section-snippet.adoc

```diff
@@ -109,4 +113,13 @@ AutoDocumentation.sectionBuilder()
     .build()
 ----
 <1> only selected snippets will be used in the specified order
-<2> if snippets having no content should be skipped from section
+<2> whether snippets having no content should be skipped from section
+
+==== Custom snippet in section
```

Comment:
"Custom snippet as a section in the section snippet" ;) But as this part of the documentation is only about the section snippet it might be clearer if we talk about adding a section (to the section snippet).